app/imports/startup/server/index.js

```diff
@@ -7,4 +7,25 @@ import './register-api.js';
 
 import './register-harvests-api.js';
 
-import './register-organizations-api.js';
\ No newline at end of file
+import './register-organizations-api.js';
+
+// Google Analytics:
+import { Inject } from 'meteor/meteorhacks:inject-initial';
+
+let gaInit = '';
+if (Meteor.settings.public.mode !== 'test') {
+    process.env.googleAnalyticsID = 'UA-78530076-8';
```

Comment:
can this be set as an environment variable instead of hardcoded?

Reply:
Yes, missed that, thought I had changed it. Will make that change. Ty.